app/helpers/pg_hero/home_helper.rb

```diff
@@ -26,5 +26,15 @@ def pghero_remove_index(query)
       ret << ", column: #{columns.inspect}" if columns
       ret
     end
+
+    def formatted_vacuum_times(time)
```

Comment:
Let's prefix helpers with `pghero_` since I think these will be global.

Reply:
Good call! Updated